src/test/java/io/anserini/collection/Iso19115CollectionTest.java

```diff
@@ -0,0 +1,33 @@
+package io.anserini.collection;
+
+import org.junit.Before;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+public class Iso19115CollectionTest extends DocumentCollectionTest<Iso19115Collection.Document> {
+  @Before
+  public void setUp() throws Exception {
+    super.setUp();
+    collectionPath = Paths.get("src/test/resources/sample_docs/Iso19115");
+    collection = new Iso19115Collection(collectionPath);
+    Path segment = Paths.get("src/test/resources/sample_docs/Iso19115/output.json");
```

Comment:
I think we want lowercase `i` to be consistent with everything else in that directory?